lib/common/logging.c

```diff
@@ -978,10 +972,24 @@ crm_log_output_fn(const char *file, const char *function, int line, int level, c
 void
 pcmk__cli_init_logging(const char *name, unsigned int verbosity)
 {
-    crm_log_cli_init(name);
+    crm_log_init(name, LOG_ERR, FALSE, FALSE, 0, NULL, TRUE);
 
     for (int i = 0; i < verbosity; i++) {
         /* These arguments are ignored, so pass placeholders. */
         crm_bump_log_level(0, NULL);
     }
 }
+
+// Deprecated functions kept only for backward API compatibility
+
+gboolean crm_log_cli_init(const char *entity);
+
+/*!
+ * \deprecated Use pcmk__cli_init_logging instead
```

Comment:
Actually we should say something like "Do not use Pacemaker for logging" -- this is the public API documentation, and external code won't have access to pcmk__cli_init_logging(). Eventually we'll deprecate the entire public logging API.

Reply:
Also we only need the doxygen block in one place (logging.h or logging.c)

Reply:
Updated in the header file.